CONTRIBUTING.md

```diff
@@ -9,3 +9,8 @@ $ gofmt -w ./
 $ staticcheck ./...
 $ go test -race ./...
 ```
+
```

Comment:
Change line 8 to `go fmt ./...` just to ensure it is inline with pre-commit checks?

Reply:
I think its a good idea!